schemas/common/axis.cue

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+#y_axis: {
```

Comment:
ah I'm wondering if it makes sense it is not related to the timeseries chart actually. Or do you foresee we will use it for another chart ? 

@AntoineThebaud I think we should use `close` here right ? Just so it won't be possible to add more unknown field there.
Also maybe the migration script needs to be updated. WDYT @AntoineThebaud ?

Reply:
SGTM I just moved it to time-series.cue, if we need it in another chart later, we can always move it to common then

Reply:
No it's ok like this @Nexucis, actually to have this closed struct restriction you have to options:

1. use the close() function
```cue
a: close({
	a1: number
	a2?: string
})
```
2. use a definition
```cue
#a: {
	a1: number
	a2?: string
}
a: #a
```

I used close() in some places because I found it less cumbersome for those, but most of the time you should use the definitions syntax (especially if you reuse it in different places)